kroxylicious-operator/pom.xml

```diff
@@ -322,4 +322,143 @@
             </testResource>
         </testResources>
     </build>
+
+    <profiles>
+        <profile>
+            <id>kubeTest</id>
+            <properties>
+                <container.tool>podman</container.tool> <!-- override for docker -->
+                <container.tag>test</container.tag>
+                <image.archive.file>/tmp/myimage.tar</image.archive.file>
+            </properties>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>exec-maven-plugin</artifactId>
+                        <version>3.5.0</version>
+                        <executions>
+                            <execution>
+                                <!-- build the container image -->
+                                <id>podman-build</id>
+                                <goals>
+                                    <goal>exec</goal>
+                                </goals>
+                                <phase>package</phase>
+                                <configuration>
+                                    <workingDirectory>..</workingDirectory>
+                                    <executable>${container.tool}</executable>
+                                    <arguments>
+                                        <argument>build</argument>
+                                        <argument>--file=./Dockerfile.operator</argument>
+                                        <argument>--build-arg=KROXYLICIOUS_VERSION=${project.version}</argument>
+                                        <argument>--tag=kroxylicious.io/operator:${container.tag}</argument>
```

Comment:
Unlike the approach taken in the `run-operator.sh` I'm using a fixed image name. Because maven's not that clever.